docs/install-docker-quickstart.md

```diff
@@ -1,264 +1,84 @@
-# Docker quickstart
+# Docker Quickstart
 
-To start developing with Nakama, you’ll first need to install it on your development machine. It’s straightforward and takes just a few minutes. In this guide we’re going to focus on installing your Nakama development instance using Docker.
+[Docker](https://www.docker.com/) is the quickest way to download and start developing with Nakama. By using Docker you are able to:
 
-!!! summary "Recommended Approach"
-    Docker is the quickest way to download and get started with Nakama for development purposes. For production settings, we recommend that you install Nakama as a [binary](install-binary.md) to ensure all system resources are available to Nakama.
+* Install to a pristine environment
+* Concurrently install and run the required [CockroachDB](https://www.cockroachlabs.com/) database
+* Take snapshots, remove, and re-install Nakama without affecting your primary operating system
+* Enjoy a quick and simplified installation experience regardless of your OS
 
-There is a single, minimal Nakama image that contains the Nakama binary. The basic format is:
+Following this guide, you will pull the required Docker images and run a local developement instance of Nakama.
 
-=== "Shell"
-	```shell
-	docker run heroiclabs/nakama <command> [options]
-	```
+## Prerequisites
 
-!!! note "Database server"
-    Nakama requires a database server running. Make sure you start the database before Nakama, or [use docker-compose to run both](#running-nakama-with-docker-compose).
+Before proceeding ensure that you have [installed Docker Desktop](https://docs.docker.com/get-docker/).
```

Comment:
Could you add a note that you need x, y, z operating system. For example Win 10 Home is not supported and going down that path might frustrate someone new to Nakama.

Reply:
Yep agreed.

Reply:
Added a note about Linux installs. Why do you say that Win 10 Home isn't supported: 
<img width="701" alt="Screen Shot 2021-05-12 at 10 29 47 AM" src="https://user-images.githubusercontent.com/83644514/117936041-0e40b300-b30d-11eb-92f9-1a34301b1cc0.png">

Reply:
Windows 10 Home doesn't have Hyper-V which is needed for Docker or WSL

Reply:
No longer the case with WSL 2: https://www.docker.com/blog/docker-desktop-for-windows-home-is-here/

Reply:
Is that out of Windows Insider preview? I tried it a couple weeks ago and wasn't successful.

Reply:
After some debugging and changing a BIOS setting, I got wsl-2 and Docker Desktop running on WIndows 10 Home. Thanks @gpene 🙏